### ISSUE TITLE
'MetaDataLoader' object has no attribute 'sample_size'

### ISSUE THREAD

**Issue by non-knife** [nuster1128/RecBole-MetaRec#1]
你好，我在运行代码的时候出现了这个错误，这个问题有什么解决的方法吗？

**Comment by nuster1128** [comment]
Hello. In RecBole-MetaRec, we do not define or use any attribute called 'sample_size'. However, when implementing your own model, it is suggested to use the utils that we provided in RecBole-MetaRec. When using original utils/modules in RecBole, please refer to [RecBole 2.0](https://github.com/RUCAIBox/RecBole2.0) to get more information. Maybe the problem is caused by incompatible modules, or it may be caused by running non-meta models that is not provided in this package.

**Comment by non-knife** [comment]
> Hello. In RecBole-MetaRec, we do not define or use any attribute called 'sample_size'. However, when implementing your own model, it is suggested to use the utils that we provided in RecBole-MetaRec. When using original utils/modules in RecBole, please refer to [RecBole 2.0](https://github.com/RUCAIBox/RecBole2.0) to get more information. Maybe the problem is caused by incompatible modules, or it may be caused by running non-meta models that is not provided in this package.

RecBole中的abstract_dataloader.py文件中有sample_size这个属性，他只在下面这段代码中出现了，其他地方好像没有，不太清楚他的作用

>    def __init__(self, config, dataset, sampler, shuffle=False):
>         self.shuffle = shuffle
>         self.config = config
>         self._dataset = dataset
>         self._sampler = sampler
>         self._batch_size = self.step = self.model = None
>         self._init_batch_size_and_step()
>         index_sampler = None
>         self.generator = torch.Generator()
>         self.generator.manual_seed(config["seed"])
>         self.transform = construct_transform(config)
>         if not config["single_spec"]:
>             index_sampler = torch.utils.data.distributed.DistributedSampler(
>                 list(range(self.sample_size)), shuffle=shuffle, drop_last=False
>             )
>             self.step = max(1, self.step // config["world_size"])
>             shuffle = False
>         super().__init__(
>             dataset=list(range(self.sample_size)),
>             batch_size=self.step,
>             collate_fn=self.collate_fn,
>             num_workers=config["worker"],
>             shuffle=shuffle,
>             sampler=index_sampler,
>             generator=self.generator,
>         )

**Comment by non-knife** [comment]
已解决，换用recbole1.0.1版本
